hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/commit/magic/MagicS3GuardCommitter.java

```diff
@@ -22,9 +22,11 @@
 import java.util.Collections;
 import java.util.List;
 
+import org.apache.hadoop.fs.s3a.commit.files.PendingSet;
```

Comment:
don't like imports to be moved around. As well as a nominal rule of layout, changes here make merging a nightmare